conf/broker.conf

```diff
@@ -833,6 +833,9 @@ maxHttpServerConnections=2048
 # Max concurrent web requests
 maxConcurrentHttpRequests=1024
 
+# enable compression metrics data when the HTTP service responds to the client
+enableCompressMetricsData=false
```

Comment:
compressOutputMetricsInPrometheus=false

Just make the name consistent with the existing flags of prometheus

And please move it under the `### --- Metrics --- ###` section